pytravharv/__init__.py

```diff
@@ -14,7 +14,7 @@
     TravHarvConfig,
 )
 from pytravharv.TravHarvExecuter import TravHarvExecutor
-from pytravharv.__main__ import TravHarv
+from pytravharv.pytravharv import TravHarv
```

Comment:
better to call this file trav_harv? instead of repeating the package name -> from pytravharv.trav_harv import TravHarv